src/Development/IDE/Core/Compile.hs

```diff
@@ -481,7 +474,7 @@ getModSummaryFromImports fp contents = do
         -- To avoid silent issues where something is not processed because the date
         -- has not changed, we make sure that things blow up if they depend on the date.
                 , ms_hsc_src      = sourceType
-                , ms_hspp_buf     = Nothing
+                , ms_hspp_buf     = Just contents
```

Comment:
Do we need this for anything now?

Reply:
Yes. The `ModSummary` rule uses the contents to extract the pragmas which are then recorded in the early cutoff hash